framework/base/Security.php

```diff
@@ -450,6 +446,11 @@ public function generateRandomKey($length = 32)
             throw new InvalidParamException('First parameter ($length) must be greater than 0');
         }
 
+        // always use random_bytes() if it is available
```

Comment:
I struggled with this problem too but decided it was better to do what PHP 7 does in PHP 7. But I guess we cannot honor to our API spec otherwise.